static/window/styles/fancy.css

```diff
@@ -426,7 +424,7 @@ body {
 
 /* ------ About ------ */
 .about-page__section__title {
-  background-color: var(--layout__tertiary-background);
+  /* background-color: var(--layout__tertiary-background); */
```

Comment:
Have these been left in on purpose?

Reply:
In all honesty, I don't know why I kept them lol. If it's a problem feel free to get rid of them.